scripts/tests/test_check_crossref_dois.py

```diff
@@ -0,0 +1,74 @@
+# encoding: utf-8
+import os
+import mock
+import pytest
+import json
+from datetime import timedelta
+import responses
+HERE = os.path.dirname(os.path.abspath(__file__))
+
+
+from osf_tests.factories import PreprintFactory
+from website import settings, mails
+
+from scripts.periodic.check_crossref_dois import check_crossref_dois, report_stuck_dois
+
+
+@pytest.mark.django_db
+class TestCheckCrossrefDOIs:
```

Comment:
These tests don't work even after fixing the import error; clearly they aren't being run by TCI. Given that we plan on running this cron, it would probably be good to have these tests actually run, unlike the rest of the `scripts` directory. It might make sense to put them in an `osf_tests/test_management_commands.py`.
```
================================================================================================================================================================================= FAILURES =================================================================================================================================================================================
_______________________________________________________________________________________________________________________________________________________________ TestCheckCrossrefDOIs.test_report_stuck_dois _______________________________________________________________________________________________________________________________________________________________
scripts/tests/test_check_crossref_dois.py:74: in test_report_stuck_dois
    to_addr=settings.OSF_SUPPORT_EMAIL)
../../.pyenv/versions/2.7.10/envs/osf/lib/python2.7/site-packages/mock/mock.py:937: in assert_called_with
    six.raise_from(AssertionError(_error_message(cause)), cause)
../../.pyenv/versions/2.7.10/envs/osf/lib/python2.7/site-packages/six.py:737: in raise_from
    raise value
E   AssertionError: Expected call: send_mail(email_content='DOIs for the following preprints have been pending at least 2 days: guid0', mail=<website.mails.mails.Mail object at 0x113ce0610>, pending_doi_count=1, to_addr='fake-support@osf.io')
E   Actual call: send_mail(guids=u'guid0', mail=<website.mails.mails.Mail object at 0x113ce0610>, pending_doi_count=1, time_since_published=2, to_addr='fake-support@osf.io')
---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- Captured log call -----------------------------------------------------------------------------------------------------------------------------------------------------------------------------
check_crossref_dois.py      85 INFO     There were 1 stuck registrations for CrossRef, email sent to help desk
______________________________________________________________________________________________________________________________________________________________ TestCheckCrossrefDOIs.test_check_crossref_dois ______________________________________________________________________________________________________________________________________________________________
<string>:3: in wrapper
    ???
scripts/tests/test_check_crossref_dois.py:62: in test_check_crossref_dois
    assert stuck_preprint.identifiers.count() == 1
E   assert 0 == 1
E    +  where 0 = <bound method GenericRelatedObjectManager.count of <django.contrib.contenttypes.fields.GenericRelatedObjectManager object at 0x11709a390>>()
E    +    where <bound method GenericRelatedObjectManager.count of <django.contrib.contenttypes.fields.GenericRelatedObjectManager object at 0x11709a390>> = <django.contrib.contenttypes.fields.GenericRelatedObjectManager object at 0x11709a390>.count
E    +      where <django.contrib.contenttypes.fields.GenericRelatedObjectManager object at 0x11709a390> = <Preprint: Organized zero tolerance help-desk (published preprint) (guid=ebgfx)>.identifiers
```

Reply:
:(